### DISCUSSION TITLE
Two-Up (Cover Page)

### DISCUSSION THREAD

**Discussion by strajk-** [QuestPDF/QuestPDF#295] [Q&A] [answered]
Haven't found anything in the docs, but is it possible to set the Page Layout of the document to Two-Up (Cover Page) like we can do with Adobe Reader in the PDF Properties?

First page would be shown by itself, every page after would then be shown side by side, as if you're reading a newspaper.
Pic related:
![image](https://user-images.githubusercontent.com/7372902/181086561-731dd537-b09c-490c-8810-b91defd394c1.png)

Currently I'm just doubling the width of the page and adding them side by side, just wondering if it is possible to se those properties.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Setting PDF viewer preferences such as page layout mode (e.g. Two-Up, Cover Page) is not currently supported by QuestPDF and is not on our roadmap. We appreciate the feedback, as community demand helps us shape future priorities.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
